inc/saimacsec.h

```diff
@@ -645,6 +645,23 @@ typedef enum _sai_macsec_sc_attr_t
      */
     SAI_MACSEC_SC_ATTR_SA_LIST,
 
+    /**
+     * @brief Cipher suite for this Secure Channel.
+     *
+     * @type sai_uint64_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_MACSEC_SC_ATTR_MACSEC_CIPHER_SUITE,
+
+    /**
+     * @brief True means encryption is enabled.  False means encryption is disabled.
+     *
+     * @type bool
+     * @flags CREATE_ONLY
+     * @default true
+     */
+    SAI_MACSEC_SC_ATTR_ENCRYPTION_ENABLE,
```

Comment:
This is definitely more optimal.  However, this change should be accompanied by removal of SAI_MACSEC_SA_ATR_ENCRYPTION_ENABLE, if the associate driver and NOS code churn is acceptable.

Reply:
Yes, I would propose to eliminate the existing SAI_MACSEC_SA_ATTR_ENCRYPTION_ENABLE in favor of SC attr.